src/Integration/MefServices/MefSonarQubeService.cs

```diff
@@ -56,8 +56,8 @@ public LoggerAdapter(ILogger logger)
             }
 
             public void Debug(string message) =>
-                // This will be executed only on DEBUG build.
-                logger.LogDebug($"DEBUG: {message}");
```

Comment:
The `LogDebug` method now prefixes all strings with `DEBUG` so it isn't needed here.